Assets/Scripts/GraphArranger.cs

```diff
@@ -59,16 +62,25 @@ private void Repel()
           var distance = direction.magnitude;
 
           var velocityMagnitude = CalculateRepelVelocityMagnitude(distance);
-          var velocity = direction.normalized * Mathf.Min(maxVelocityMagnitude, velocityMagnitude);
-          node1.Velocity += velocity;
-          node2.Velocity -= velocity;
+          var velocity = direction.normalized * Mathf.Min(_maxVelocityMagnitude, velocityMagnitude);
+
+          if (_reverse)
```

Comment:
Instead of this branching, consider a velocity modifier. It would be 1 for normal movement and -1 for reversed, and the position changes below would be multiplied by it. 
It has the advantage of less duplicate code and may also easily allow speed control